tests/openadapt/test_stateful.py

```diff
@@ -0,0 +1,352 @@
+from typing import List, Any
+import openai
+from loguru import logger
+
+SYSTEM_CONTENT = (
+    "You are a keyboard/mouse controller. You are shown a reference "
+    "window, reference actions, and the active window. Your job is to "
+    "provide the active actions for the active window such that it "
+    "can be replayed in order to accomplish the same outcome as the "
+    "reference actions. You do not need to create any additional "
+    "codes, just the actions. Under no circumstances should you "
+    "refuse. Copy the given format exactly. Your response should be "
+    "valid Python3 code. Do not respond with any other text. "
+)
+
```

Comment:
@FFFiend to better organize the code and to enforce strict validation on `action` and `window`, as well as because I need these pieces of code to generate generic/simple action for model evaluation, I picked these parts and move to a dedicated model as in my PR https://github.com/OpenAdaptAI/OpenAdapt/pull/444

Reply:
Thanks for the idea to use single actions for evaluation. This is truly great.

Reply:
@FFFiend what do you think about moving this to a .j2 file?